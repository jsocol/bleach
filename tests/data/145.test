<div id="74"><a href="javascript:alert(74)"><event-source src="data:application/x-dom-event-stream,Event:click%0Adata:XXX%0A%0A" /></a>//["'`-->]]>]</div>
