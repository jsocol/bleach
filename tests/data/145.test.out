<div id="74"><a></a>//["'`--&gt;]]&gt;]</div>
